build.sbt

```diff
@@ -49,6 +49,7 @@ lazy val flows = setupProject(
 lazy val source_model = setupProject(
   project,
   folder = "common/source_model",
+  localDependencies = Seq(internal_model),
```

Comment:
Is this a bad coupling? It feels uncomfortable

Reply:
Yeah, I don't love it – but I couldn't think of a better way to do this that doesn't involve passing implicit knowledge of the internal models around. 😕